src/meta/Drag.ts

```diff
@@ -0,0 +1,170 @@
+import global from '@dojo/shim/global';
+import { assign } from '@dojo/shim/object';
+import WeakMap from '@dojo/shim/WeakMap';
+import { Base } from './Base';
+
+export interface DragResults {
+	/**
+	 * The movement of pointer during the duration of the drag state
+	 */
+	delta: Position;
```

Comment:
nit: should we declare the interface before we use it? (I thought it was an from an import)

Reply:
I was favouring alphabetisation over use before declaration... one form of clarity causes another issue

Reply:
I guess so.